src/core/helper.js

```diff
@@ -0,0 +1,10 @@
+export const associateTileToBackground = grid => {
+    const tileSize = 100 / grid.length; // 100%
+    let tileToBg = {};
+    grid.forEach((row, rowKey) => {
```

Comment:
Prefer reduce use

Reply:
I find that use two reduce is less readable.